### DISCUSSION TITLE
Parallax Experiment

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#302] [Show and tell]
https://github.com/keiwando/toonsquid/assets/129681998/a67b90e1-f73c-4bb5-93ea-8cb9798a2462

This here is an experiment I did trying to figure out how to do a parallax effect to create more three dimensional backgrounds. As far as I know, there isn't a set formula for how fast each layer should be when doing something like this. I basically used Procreate to create the perspective grid.

**Comment by josecm1** [comment]
Far away objects should move slowly, or not moving at all like very far mountains. Planes near to the camera moves faster.

**Reply by DWrightAnimation** [reply]
I'm not sure what process they do in cartoons to make it look good, but I'm guessing they start in the middle, and animate the foreground and background elements afterwards. Either way, this was just an experiment on my part, and I hope to improve on future experiments.

**Reply by DinoAwakening** [reply]
> I'm not sure what process they do in cartoons to make it look good, but I'm guessing they start in the middle, and animate the foreground and background elements afterwards. Either way, this was just an experiment on my part, and I hope to improve on future experiments.

They usually just use Mathematics. More specifically, power. So, they just multiply the pixels moved per frame (or even frames itselves) by 2, 3, 4 or even 8x slower for further Background. 🤓☝️
